.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -12,6 +12,14 @@ jobs:
         CONFIG: linux_64_
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
+      linux_aarch64_:
+        CONFIG: linux_aarch64_
+        UPLOAD_PACKAGES: 'True'
+        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
```

Comment:
The docker image is `x86_64` even for `aarch64` (and `ppc64le` below)? Is the docker image for the host system?

Same for the individual `.yaml` files in `.ci_support`.

Reply:
Yes, this is cross-compiling all new architecture I've added here.